src/block-management/block-toolbar.js

```diff
@@ -38,29 +38,38 @@ export class BlockToolbar extends Component<PropsType> {
 		} = this.props;
 
 		return (
-			<View style={ styles.container }>
+			<View style={ styles.container } accessible={ false } accessibilityLabel="View Add block">
 				<ScrollView
+					accessible={ false }
+					accessibilityLabel="Scrollview Add block"
 					horizontal={ true }
 					showsHorizontalScrollIndicator={ false }
 					keyboardShouldPersistTaps={ 'always' }
 					alwaysBounceHorizontal={ false }
 					contentContainerStyle={ styles.scrollableContent }
 				>
-					<Toolbar>
+					<Toolbar accessible={ true }
+						accessibilityLabel={ 'Toolbar Add block' }>
 						<ToolbarButton
-							label={ __( 'Add block' ) }
+							title={ __( 'Add block' ) }
 							icon="insert"
+							accessible={ true }
```

Comment:
I am curious about one thing here. Do we need to also set accessible={ true } ? Could we run UI tests by just setting the (accessibility identifier)testID? Opening the element for accessibility features looks to me like something we shouldn't be doing unless we are really intending to, since it requires separate type of focus and testing. I wonder how other reviewers thinking on this though.

Reply:
Hmm you're right here I don't think I need it in this case, I never gave much thought as to how this affects Voiceover/Talkback or any other accessibility features, I assumed that it wasn't in the plans to support that in the sample app. Though [the documentation](https://facebook.github.io/react-native/docs/view.html#testid) recommends testID for this specific scenario but since it doesn't work on Android I just opted to stick with accessibilityLabel.

Reply:
Shared my opinions in the other comment: https://github.com/wordpress-mobile/gutenberg-mobile/pull/676#discussion_r266770238

Reply:
If we are able to set accessible={ false } and use accessibilityLabel for UI testing at the same time, we'll have the option to disable accessibility for elements we don't intend on making publicly visible. So I think it'd be useful to find this out.

Reply:
Yep we can set accessible to false and only set the accessibilityLabel, I'll update to go with that approach

Reply:
That's good news!